src/databricks/labs/ucx/assessment/queries/00_1_count_total_databases.sql

```diff
@@ -1,4 +1,4 @@
 -- viz type=counter, name=Total Database Count, counter_label=Total Databases, value_column=count_total_databases
--- widget col=0, row=0, size_x=1, size_y=3
+-- widget row=0, col=1, size_x=1, size_y=3
```

Comment:
Does the naming express the row/col position on the dashboard now? If so, why track this twice? So that they sort as files the same way? That would make sense to me.